### PR TITLE
[DDW-749] Remove Alonzo tada icon and countdown screen

### PR DESCRIPTION
**Pull request by mchappell**
This PR
- Removes the 'tada' button from the top navigation
- Removes Staking info screen from staking navigation
- Hides the Staking info screen and routing behind a configuration property

## Todos

- [x] Remove the 'tada' button from the top navigation
- [x] Remove Staking info screen from staking navigation
- [x] Hide the Staking info screen and routing behind a configuration property

## Screenshots

image

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

## Test Scenarios

**Scenario 1 - Validate the "tada" icon is not visible while syncing**
```gherkin
Given that Daedalus is syncing
And the syncing progress has reached the Alonzo era epoch
When I check the top bar
Then I can see that no "tada" icon is displayed
```

**Scenario 2 - Validate the "tada" is not visible when Daedalus is synced**
```gherkin
Given that Daedalus is synced
When I check the top bar
Then I can see that no "tada" icon is displayed
And on "Staking center" the "info" tab is no longer visible
```

_Note: Use `daedalus.api.ada.getNetworkParameters()` to get the starting epochs for all the eras._

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/containers/staking/Staking.js [hunk 1]
```diff
@@ -75,14 +76,9 @@ export default class Staking extends Component<Props> {
...
-    const {
-      isSynced,
-      syncPercentage,
-      isAlonzoPending,
-      isAlonzoActivated,
-    } = networkStatus;
+    const { isSynced, syncPercentage } = networkStatus;
     const { isStakingDelegationCountdown } = staking;
-    const shouldShowInfoTab = isAlonzoPending || isAlonzoActivated;
+    const shouldShowInfoTab = IS_STAKING_INFO_PAGE_AVAILABLE;
```
Comment (danielmain) [inline source/renderer/app/containers/staking/Staking.js:81]: is `shouldShowInfoTab` not redundant?

**Review by danielmain — CHANGES_REQUESTED** [review]
Great job @mchappell just a few details

**Review by gabriela-ponce — APPROVED** [review]

File: source/renderer/app/containers/TopBarContainer.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import menuIconOpened from '../assets/images/menu-opened-ic.inline.svg';
 import menuIconClosed from '../assets/images/menu-ic.inline.svg';
 import { matchRoute } from '../utils/routing';
 import { ROUTES } from '../routes-config';
+import { IS_STAKING_INFO_PAGE_AVAILABLE } from '../config/stakingConfig';
```
Comment (nikolaglumac) [inline source/renderer/app/containers/TopBarContainer.js:14]: @mchappell please introduce a `IS_TADA_ICON_AVAILABLE` configuration variable to be used for the icon checks 🙏

File: source/renderer/app/containers/TopBarContainer.js [hunk 3]
```diff
@@ -39,8 +40,12 @@ export default class TopBarContainer extends Component<Props> {
     } = networkStatus;
     const { stakingInfoWasOpen } = staking;
     const shouldShowTadaIconAnimation =
-      isAlonzoActivated && !stakingInfoWasOpen;
-    const shouldShowTadaIcon = isAlonzoPending || isAlonzoActivated;
+      IS_STAKING_INFO_PAGE_AVAILABLE &&
```
Comment (nikolaglumac) [inline source/renderer/app/containers/TopBarContainer.js:43]: @mchappell there is no need to add this check for the animation.

**Pull request merged**